package.json

```diff
@@ -16,6 +16,7 @@
   ],
   "dependencies": {
     "args": "^5.0.1",
+    "autoprefixer": "^9.7.0",
```

Comment:
Shouldn't we rather move this (and other modules that are only needed during the build) to devDependencies as well?

Reply:
Yeah, but this is simpler for deployments.

Reply:
We can revert to the previous behavior and move the devDeps. But if one has set production on Heroku, we need that extra step to explicitly install the devDeps.